### PR TITLE
Basic project structure and WASM module project files

### PR DESCRIPTION
**Pull request by StevenFrost**

### CODE REVIEW

File: Build/Config/Cpp.props [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+  <ItemDefinitionGroup>
+    <Link>
+      <SubSystem>Windows</SubSystem>
+      <GenerateDebugInformation>true</GenerateDebugInformation>
+      <EnableCOMDATFolding Condition="'$(Configuration)' == 'Release'">true</EnableCOMDATFolding>
+      <OptimizeReferences Condition="'$(Configuration)' == 'Release'">true</OptimizeReferences>
+    </Link>
+    <ClCompile>
+      <LanguageStandard>stdcpp14</LanguageStandard>
```
Comment (ollyau) [inline Build/Config/Cpp.props]: Do we want `stdcpp14`, or should we go with something like `stdcpp17` or `stdcpplatest`?  When I last tried, it still builds correctly with `stdcpplatest` set, but not _all_ features are enabled with the MSFS SDK version of WASI (e.g. no exceptions, no `std::filesystem`, etc.).

File: Build/Config/Cpp.props [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      <MinimalRebuild>false</MinimalRebuild>
+      <Optimization Condition="'$(Configuration)' == 'Release'">MaxSpeed</Optimization>
+      <Optimization Condition="'$(Configuration)' != 'Release'">Disabled</Optimization>
+      <BasicRuntimeChecks Condition="'$(Configuration)' == 'Release'"></BasicRuntimeChecks>
+
+      <PreprocessorDefinitions Condition="'$(Configuration)' == 'Debug'">_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(Configuration)' != 'Debug'">NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(ConfigurationType)' == 'StaticLibrary'">_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+
+      <PreprocessorDefinitions Condition="'$(Platform)' == 'Win32'">WIN32;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```
Comment (ollyau) [inline Build/Config/Cpp.props]: `Win32`?  Is this even supported with Asobo SimConnect?

**Review by ollyau — APPROVED** [review]
Looks good to me!  Feel free to merge, but I just added a couple questions re. the language standard and platform in the cpp props.

File: Build/Config/Cpp.props (hunk 2 above)
Comment (StevenFrost) [reply Build/Config/Cpp.props]: Ah, that's from the old props. I'll remove this

File: Build/Config/Cpp.props (hunk 1 above)
Comment (StevenFrost) [reply Build/Config/Cpp.props]: Switched to C++17 :)

**Review by ollyau — APPROVED** [review]
Maybe this approves it again?

**Review by ollyau — APPROVED** [review]
Extra approved!

**Pull request merged**